frontend/src/data/tasks.json

```diff
@@ -272,7 +272,7 @@
       {
         "answerKey": "between-60-and-65",
         "descriptionEn": "The CPP lets you choose when to start your retirement pension, and each month you delay increases your monthly amount.  The highest monthly amount you can receive happens at age 70, after which there is no benefit to waiting.  If you need money sooner, you can start collecting your pension as early as age 60, but with a permanent reduction.\n\nWhen you apply, you will need to choose when you want your pension to start.\n- As soon as you qualify (the month after you turn 60)\n- At a specific date that you choose (after age 60)\n\nYou can apply on paper in-person or by mail. For a faster reply, apply online. For more details, see the links below.",
-        "descriptionFr": "Le RPC vous permet de choisir quand débuter votre pension de retraite, et chaque mois que vous reportez augmente votre montant mensuel. Le montant mensuel le plus élevé que vous pouvez recevoir arrive à 70 ans, après quoi il n'y a aucun avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à toucher votre rente dès l'âge de 60 ans, mais avec une réduction permanente.\n\nLors de votre demande, vous devrez choisir la date à laquelle vous souhaitez commencer à percevoir votre pension.\n- Dès que vous remplissez les conditions requises (le mois suivant votre 60 anniversaire)\n- À une date précise que vous choisissez (après l'âge de 60 ans)\n\nVous pouvez postuler sur papier en personne ou par courrier. Pour une réponse plus rapide, postulez en ligne. Pour plus de détails, voir les liens ci-dessous.",
+        "descriptionFr": "Le RPC vous permet de choisir quand commencer votre pension de retraite, et chaque mois que vous reportez, le montant mensuel augmente. Le montant le plus élevé à recevoir est à l'âge de 70 ans. Après cet âge, il n'y a aucun avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à recevoir votre pension dès l'âge de 60 ans, mais avec une réduction permanente.\n\nLors de votre demande, vous devrez choisir la date à laquelle vous souhaitez commencer à recevoir votre pension.\n- Dès que vous remplissez les conditions requises (le mois suivant votre 60 anniversaire)\n- À une date précise que vous choisissez (après l'âge de 60 ans)\n\nVous pouvez présenter votre demande en format papier, en personne ou par la poste. Pour une réponse plus rapide, faites votre demande en ligne. Pour plus de détails, consultez ces liens.",
```

Comment:
The list items in this block do not have semicolons and a period like the others do. Was this intentional?